public/app/plugins/panel/geomap/layers/data/markersLayer.tsx

```diff
@@ -173,6 +178,55 @@ export const markersLayer: MapLayerRegistryItem<MarkersConfig> = {
               step: 0.1,
             },
           })
+          .addCustomEditor({
+            id: 'config.style.text',
+            path: 'config.style.text',
+            name: 'Text label',
+            editor: TextDimensionEditor,
+            defaultValue: defaultOptions.style.text,
+          })
+          .addNumberInput({
```

Comment:
can you work with @nmarrs  to incorporate this into #41716

bit of chicken egg though :)  Likely this should go fist then then #41716

Reply:
We probably can merge these changes first and I can through and add these additional fields to the `StyleEditor`